### PR TITLE
[issues:97981] del unrelated comment

### LINKED ISSUE #97981: Typo in `src/test/ui/variance/variance-object-types.rs`
```
// Test that Cell is considered invariant with respect to its            <- unrelated comment
// type.
```

### PR DESCRIPTION
**Pull request by dust1**
fixes https://github.com/rust-lang/rust/issues/97981
summary: del unrelated comment

### CODE REVIEW

File: src/test/ui/variance/variance-object-types.rs [hunk 1]
```diff
@@ -1,6 +1,3 @@
-// Test that Cell is considered invariant with respect to its
-// type.
-
 #![feature(rustc_attrs)]
 
 use std::cell::Cell;
```
Comment (eggyal) [inline src/test/ui/variance/variance-object-types.rs:3]: This `use` statement can go too.

**Comment by Dylan-DPC** [comment]
@dust1 why did you mark this as a draft? seems complete to me (minus the review comments)

**Comment by dust1** [comment]
> @dust1 why did you mark this as a draft? seems complete to me (minus the review comments)

Sorry, maybe I didn't read contribution doc carefully? 
I don't have much experience, another project I've PR on always asks me to use draft, I thought it was all

**Comment by Dylan-DPC** [comment]
> Sorry, maybe I didn't read contribution doc carefully?
I don't have much experience, another project I've PR on always asks me to use draft, I thought it was all

no it's fine. generally you mark it as draft when it is WIP - as in you still are working on it and need to push more changes for it to be ready for reviewing. Otherwise you just open a normal PR. 

Nothing wrong, but it took me by surprise that you went the draft route for a trivial pr :P

**Comment by eggyal** [comment]
Also just an FYI: if your commit message said "closes <issue no>" or "fixes <issue no>" rather than "issue: <issue no>" then GitHub would automatically close that issue when the PR is merged.

See https://github.blog/2013-01-22-closing-issues-via-commit-messages/

**Comment by dust1** [comment]
> Also just an FYI: if your commit message/PR description included `closes <issue no>` or `fixes <issue no>` rather than `issue: <issue no>` then GitHub will automatically close that issue when the PR is merged (I think it will still work if you edit it before merge).
> 
> See https://github.blog/2013-01-22-closing-issues-via-commit-messages/

Thanks, this experience is exactly what I lack.  :P

**Pull request merged**